TRANSLATION.md

```diff
@@ -0,0 +1,140 @@
+Translating Kibana
```

Comment:
Let's move this into `docs/development/plugin/development-internationalization.asciidoc`.